src/main/java/com/dev/cinema/controller/UserController.java

```diff
@@ -22,7 +24,7 @@ public UserController(UserService userService, UserMapper userMapper) {
     }
 
     @GetMapping("/by-email")
-    public UserResponseDto get(@RequestParam String email) {
+    public UserResponseDto get(@RequestParam @Size(min = 11) @Valid String email) {
```

Comment:
does it work without the steps listed in this article?
https://www.baeldung.com/spring-validate-requestparam-pathvariable